docs/archive/0.3.3/sql/functions/numeric.md

```diff
@@ -32,7 +32,7 @@ The table below shows the available mathematical functions.
 | `abs(x)` | absolute value | `abs(-17.4)` | 17.4 |
 | `acos(x)` | computes the arccosine of x | `acos(0.5)` | 1.0471975511965976 |
 | `asin(x)` | computes the arcsine of x | `asin(0.5)` | 0.5235987755982989 |
-| `atan2(x)` | computes the arctangent of x | `atan2(0.5)` | 0.4636476090008061 |
+| `atan2(x)` | computes the arctangent of x | `atan(0.5)` | 0.4636476090008061 |
```

Comment:
The first column should also be `atan` for these?

Reply:
Thanks @hawkfish - I've corrected the `Function` column as well.